activities/zio-flow-twilio/shared/src/main/scala/zio/flow/activities/twilio/Message.scala

```diff
@@ -0,0 +1,54 @@
+package zio.flow.activities.twilio
+
+import zio.flow._
+import zio.schema.DeriveSchema
+
+final case class Message(
+  body: String,
+  num_segments: String,
+  direction: String, // TODO: should be Direction serialized into a single string field
```

Comment:
We cannot use the `Direction` type here because by default the Json codec encodes sum types with json objects, and here it is sent as a simple string. 

If we would write our own `Schema` for `Direction` that transforms a `Schema[String]` it would work in theory but not in the context of zio-flow, where we loose the transformation when the schema get serialized. 

So if we would do that, in the executor the dynamic value representing `Message` would contain a field `direction` holding a primitive string. If the flow then does anything with this value using the optics, that will expect to have a `DynamicValue.Enumeration` and fail.

Reply:
There are tickets in ZIO Schema for that.